tests/console/zypper_info.pm

```diff
@@ -29,29 +39,35 @@ sub run() {
     my $expected_package_name_vim = 'Name *: vim';
     die "Missing package name. Expected: /$expected_package_name_vim/"
       unless $info_output_vim =~ /$expected_package_name_vim/;
+}
 
-    if (sle_version_at_least('12-SP3')) {
+sub run() {
+    select_console 'root-console';
+
+    # preparation of source repositories
+    my $cmd;
+    if (check_var('DISTRI', 'sle')) {
+        # SLE maintenance tests are assumed to be SCC registered
+        # and source repositories disabled by default
+        if (get_var('FLAVOR') =~ /-Updates$/) {
+            $cmd = q{mr -e $(zypper -n lr | awk '/-Source/ {print $1}')};
+        }
         # use dvd2 as the src-repository
-        zypper_call('ar --disable --type plaindir cd:///?devices=/dev/sr1 repo-source');
+        else {
+            $cmd = 'ar --type plaindir cd:///?devices=/dev/sr1 repo-source';
+        }
     }
-
-    if (check_var('DISTRI', 'opensuse') || sle_version_at_least('12-SP3')) {
-        test_dvd_src_as_repo();
+    # source repository is disabled by default
+    elsif (check_var('DISTRI', 'opensuse')) {
+        $cmd = "mr -e repo-source";
```

Comment:
yes, correct but it was wrong before and still is so for this PR it's ok but needs to be changed afterwards anyway.